src/Components/Utils/apis.js

```diff
@@ -0,0 +1,7 @@
+export const fetchAllArticles  = () => {
+
```

Comment:
Please use axios.create() here so you will have a base URL which can be used for all your request functions. (See NC Notes/Axios bottom of the page)